assets/component-card.css

```diff
@@ -1,290 +1,149 @@
-.card {
-  display: block;
-  overflow: hidden;
+.card-wrapper {
+  color: inherit;
+  height: 100%;
+  position: relative;
   text-decoration: none;
 }
 
-.card-information + .card {
-  margin-bottom: 1.3rem;
-}
-
-@media screen and (min-width: 750px) {
-  .card-information + .card {
-    margin-bottom: 1.7rem;
-  }
-}
-
-.card.card--soft {
-  background-color: rgba(var(--color-foreground), 0.04);
-  color: rgb(var(--color-foreground));
-}
-
-.card .icon-wrap {
-  margin-left: 0.8rem;
-  white-space: nowrap;
-  transition: transform var(--duration-short) ease;
-  overflow: hidden;
-}
-
-.card--media {
-  display: block;
+.card {
+  text-align: var(--card-text-alignment);
   text-decoration: none;
 }
 
-.card--text-only {
-  display: flex;
-  justify-content: center;
-}
-
-.card--text-only.card--product {
-  position: static;
-}
-
-.card--text-only::before {
-  content: '';
-  display: block;
-  padding-bottom: 100%;
-}
-
-.card--product {
+.card--card,
+.card--standard .card__inner { 
+  background-clip: padding-box;
+  border-radius: var(--card-corner-radius);
+  border: var(--card-border-width) solid rgba(var(--color-foreground), var(--card-border-opacity));
+  box-shadow: var(--card-shadow-horizontal-offset) var(--card-shadow-vertical-offset) var(--card-shadow-blur-radius) rgba(var(--color-foreground), var(--card-shadow-opacity));
+  overflow: hidden;
   position: relative;
 }
 
-.card--stretch {
-  height: 100%;
-}
-
-.card--outline:not(.card--soft) {
-  border: calc(0.1rem / var(--font-body-scale)) solid rgba(var(--color-foreground), 0.04);
+.card--standard.card--text {
+  background-color: transparent;
 }
 
-.card--light-border {
-  border: 0.1rem solid rgba(var(--color-foreground), 0.04);
+.card-information {
+  text-align: var(--card-text-alignment);
 }
 
-.card--light-border:hover {
-  border: 0.1rem solid rgba(var(--color-foreground), 0.3);
-  box-shadow: none;
+.card__media,
+.card .media {
+  bottom: 0;
+  position: absolute;
+  top: 0;
 }
 
-.card__text-spacing {
-  padding: 3rem;
+.card .media {
+  width: 100%;
 }
 
-.card-colored.color-background-1 {
-  background-color: rgba(var(--color-foreground), 0.04);
+.card__media {
+  margin: var(--card-image-padding);
+  width: calc(100% - 2 * var(--card-image-padding));
 }
 
-.card--media .card__text-spacing {
-  padding: 2rem;
+.card--standard .card__media {
+  margin: var(--card-image-padding);
```

Comment:
Maybe we could apply a border-radius to the media in this situation ? 🤔  
![](https://screenshot.click/21-11-3ijhu-kta4m.png)